src/audio/google/Kconfig

```diff
@@ -16,6 +16,7 @@ config COMP_GOOGLE_RTC_AUDIO_PROCESSING
 	bool "Google Real Time Communication Audio processing"
 	select COMP_BLOB
 	select GOOGLE_RTC_AUDIO_PROCESSING_MOCK if COMP_STUBS
+	depends on IPC_MAJOR_4
```

Comment:
@cujomalainey, @andyross I think this is how this PR excluded  `google_rtc_audio_processing.c` from the compilation check: 

https://github.com/thesofproject/sof/actions/runs/6815375008/job/18534330110?pr=8469

```
warning: COMP_GOOGLE_RTC_AUDIO_PROCESSING (defined at
/home/runner/work/sof/sof/workspace/sof/zephyr/../src/audio/google/Kconfig:15) was assigned the
value 'y' but got the value 'n'. Check these unsatisfied dependencies: IPC_MAJOR_4 (=n)
```

`CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=y` is in `app/stub_build_all_ipc3.conf` but not in `app/stub_build_all_ipc4.conf`. I've noticed other differences between these two .conf files; you may want to bring them somewhat back in sync again.